plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -330,8 +330,10 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get", "update", "patch", "delete").Groups(legacyGroup).Resources("endpoints").Names("kube-scheduler").RuleOrDie(),
 
 				// fundamental resources
-				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes", "pods").RuleOrDie(),
+				rbac.NewRule(Read...).Groups(legacyGroup).Resources("nodes").RuleOrDie(),
+				rbac.NewRule(append(Read, "delete")...).Groups(legacyGroup).Resources("pods").RuleOrDie(),
 				rbac.NewRule("create").Groups(legacyGroup).Resources("pods/binding", "bindings").RuleOrDie(),
+				rbac.NewRule("create").Groups(legacyGroup).Resources("pods/eviction", "evictions").RuleOrDie(),
```

Comment:
are `evictions` a top-level resource? I hadn't seen them before. I expected just `pods/eviction` here

Reply:
Actually, this was a remnant of an earlier version of the code which was using eviction. More recent versions do not use evictions. So, I deleted this.